guidelines/ambassador-allontanamento.md

```diff
@@ -0,0 +1,34 @@
+
+Quando viene usato il termine _sondaggio standard infra-gruppo_ si intende un sondaggio con quorum al 50% e maggioranza semplice della durata di 48 ore tra i membri del gruppo di riferimento.
+
+Il mancato raggiungimento del quorum in un sondaggio standard infra-gruppo corrisponderà all'apertura del medesimo sondaggio, con le medesime condizioni, rivolto a tutti i membri del team Ambassador.
+
+Un membro del team Ambassador può essere allontanato per i seguenti motivi:
+
+- Mancato rispetto del [Codice di Comportamento](https://github.com/Il-Libro-Open-Source/book/blob/main/CODE_OF_CONDUCT.md). In tal caso, verrà richiesto un _sondaggio standard infra-gruppo_ nel Governance Group.
+- Violazione delle linee guida imposte dal Communications Group all'interno del gruppo Telegram. In tal caso, verrà richiesto un _sondaggio standard infra-gruppo_.
+- Violazione delle linee guida imposte dal Relations Group all'interno del perimetro di azione di quest'ultimo (Conferenze, post pubblici, ecc.). In tal caso, verrà richiesto un _sondaggio standard infra-gruppo_.
+- Non partecipazione ad almeno il 50% dei sondaggi proposti in un periodo di 30 giorni senza giustificati motivi. In tal caso, sarà il gruppo stesso a decidere se allontanare o meno la figura con un sondaggio tra i propri membri con quorum al 50% e maggioranza semplice della durata di 48 ore.
+- Mancata assegnazione di un capitolo entro 30 giorni dalla data di nomina. In tal caso, verrà fatta una comunicazione da parte del _Governance Group_ alla figura interessata, che avrà 24 ore di tempo per assegnarsi un capitolo.
+- Mancata stesura di contenuti propri o review di contenuti altrui nei capitoli assegnati entro 60 giorni dall'ultima stesura e/o review. In tal caso, verrà fatta una comunicazione da parte del _Governance Group_ alla figura interessata, che avrà 24 ore di tempo per dare una motivazione plausibile. A risposta ricevuta, verrà richiesto un _sondaggio standard infra-gruppo_ per la sua rimozione. Questa regola non si applica ai membri del team Ambassador assegnati a capitoli che non hanno ricevuto alcuna proposta di modifica.
+
+## Indice di gradimento
+
+Ogni trimestre verrà effettuato un sondaggio contenente l'intera lista dei membri del team Ambassador.  
+Il sondaggio sarà anonimo, a risposta multipla e avrà una durata di 15 giorni.  
+Qualora un membro del team Ambassador non raggiunga il 50% di gradimento, verrà allontanato dal team.
+La mancanza di votazione in questo sondaggio senza giustificati motivi verrà considerata come mancanza di partecipazione e verrà aperto un _sondaggio standard infra-gruppo_ nel Governance Group.
+
+La motivazione di questa regola è quella di garantire un team sempre attivo e partecipe, che riduca al minimo gli attriti e che possa valutare l'impatto di ogni suo membro ed esprimere il proprio giudizio a prescindere dalle linee guida che possono essere facilmente aggirate.
```

Comment:
Questa motivazione non mi fa impazzire ad essere sincero... potrebbe essere usato nel modo sbagliato. Uno può essere antipatico o di carattere forte, ma se contribuisce e fa il suo non mi sembra corretto nei confronti della persona allontanarlo solo perchè in modo anonimo sta antipatico a qualcuno...

Alla fine se non viola niente del regolamento o del CoC è gisto che resti!

Aggiungo che potrebbe creare secondo me una community che decide per antipatie o simpatie e creando quindi una "casta", passatemi il termine.

Reply:
Questo rimane un po' un nodo da capire bene, nel senso che in generale sono d'accordo con te, ma non vorrei trovarmi nella situazione per cui "Siamo in 50, a tutti noi non sta bene il modo di comportarsi di 1 di noi, ma non violando il CoC rimane", anche se vista l'attuale formazione dubito sia una situazione che accadrà 🚀

Reply:
Capisco il punto di vista, però forse implicitamente violi la CoC, comunque probabilmente si potrebbe anche pensare che si possano mandare delle segnalazioni e il wg cercherà di sistemare questi attriti nel caso prima che sfocino e peggiorino!

Reply:
A questo punto ha senso che il Governance Group abbia potere di emettere un sondaggio di "sfiducia" in qualche modo.